apps/backend/package.json

```diff
@@ -14,26 +14,33 @@
     ]
   },
   "main": "app.js",
+  "engines": {
+    "node": "^20",
+    "yarn": "^4.5.0"
+  },
+  "packageManager": "yarn@4.5.0",
   "scripts": {
-    "start": "yarn workspace @hylo/shared build:cjs && node --optimize_for_size --max_old_space_size=$MAX_OLD_SPACE_SIZE --gc_interval=100 app.js",
```

Comment:
What was coming-up with this on-the-fly transpire on deploy?

Reply:
yep, I did peripheral research after your note yesterday, and that tracks. it seems heroku doesn't install dev dev dependencies on build--which makes entire sense. we basically have to add typescript to regular dependencies--which is weird/wrong--or we look at other solutions on a few heroku threads/tickets out there. it is a common requirement / question, but also if I'm understanding what you have now with the postbuild hook, that may just be the solution.